STEERING-COMMITTEE.md

```diff
@@ -1,27 +1,108 @@
+build and test infrastructure, web sites and their domains, blogs, and social
+media accounts.
+1. Providing neutral mediation as appropriate to try to resolve non-technical
+disputes that arise as part of the project.
+
+## Membership and voting
+1.  The Steering Committee is structured to allow companies who are most
+invested in the success of the Istio project to participate in business and
+non-technical decision-making.
+1.  All members should abide by the project Code of Conduct.
+1.  There are two types of seats on the Steering Committee: Contributor Seats
+and Community Seats.
+1.  Both Contributor and Community Seats will be appointed beginning in
+July 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1.  Contributor Seat terms expire on January 1 and Community Seat terms expire
+on July 1. If necessary, a company holding a Contributor Seat may change the
+appointed individual at any time during the term.
+1.  No Contributing Company can have more than 5 seats in total on the Steering
+Committee.
+    1.  Subsidiaries, an entity in which another company owns more than 50%
+    of the voting or membership interests; and related companies, companies
+    which are controlled by a common third party entity; and affiliate
+    companies, will be treated together as a single entity, referred to as a
+    Contributing Company.
+1.  There shall be nine **Contributor Seats**. Their allocation is determined
+by the approximate effort and expenditure on the Istio project, as approximated
+by the number of merged pull requests on GitHub over the one year period prior
+to the Contributor Seat assignments:
+    1.  The top three contributing companies to Istio are eligible for
+    Contributor Seats, proportional to their contribution.
+        1.  Each company is allocated one seat;
+        1.  The remaining six seats are allocated based on percentage project
+        contribution, with no Contributing Company exceeding 5 seats in total
+        as outlined in section 6.
+1.  There shall be four **Community Seats.**
+    1.  Two **Community Seats** will be elected by the Istio contributors and
+    community, beginning in July 2020.
+        1.  Any [project member](ROLES.md#member) can self-nominate for the
+        election, or nominate another project member with their consent.
+        1.  Elections use time-limited, Condorcet voting.
+        1.  The following are eligible to vote for Community Seats:
+            1.  Project [Members](ROLES.md#member) who have had a merged PR in
+            the 12 months prior to the election; and
+            1.  People who have submitted a voting exception form to the
+            Steering Committee, demonstrating contribution to the Istio project
+            that is of a non-code nature in the 12 months prior to the election,
```

Comment:
You should allow the SC to delegate this responsibility to an assigned community member or committee.  Otherwise, you're in for some really long meetings.

Reply:
Oh! This voting will occur electronically to avoid long meetings for election purposes! Kubernetes utilizes [CIVS](http://civs.cs.cornell.edu/) for this; we will likely use the same or similar.

Reply:
The only thing Steering has to do by vote is allocate standing to non-members. We don't expect a lot of this.